app/src/main/java/com/github/se/eventradar/model/User.kt

```diff
@@ -0,0 +1,16 @@
+package com.github.se.eventradar.model
+
+data class User(
```

Comment:
question: was this checked with @christinehahaa?

Reply:
yes!

Reply:
Out of curiosity, could you explain what are the fields 
- eventsAttendeeList & eventsHostList : are you storing the events in which the user is part of / is a host?
- qrCodeUrl (what is the qr code used for exactly?)

Reply:
Yes, the eventsAttendeeList and eventsHostList fields store the IDs of events that the user is attending or hosting, respectively. 
The qrCodeUrl is linked to a unique QR code for each user, which serves multiple purposes: it acts as a digital ticket for events—allowing event staff to scan it to verify the user's ticket purchase—and it can also be used to quickly add the user as a friend by scanning it with a mobile device.